test/util/spec.helpers.js

```diff
@@ -0,0 +1,117 @@
+var helpers = require('../../lib/util/helpers');
+
+describe('helpers', function () {
+
+    describe('getPreviousSteps', function () {
+        var getPreviousSteps, steps;
+
+        beforeEach(function () {
+            getPreviousSteps = helpers.getPreviousSteps;
+        });
+
+        it('is a function', function () {
+            getPreviousSteps.should.be.a('function');
+        });
+
+        it('returns an array', function () {
+            getPreviousSteps().should.be.an('array');
+        });
+
```

Comment:
@joefitter I've done this to sense check what I was saying, https://github.com/UKHomeOffice/passports-form-wizard/compare/feature/fork-backlinks